### ISSUE TITLE
script error

### ISSUE THREAD

**Issue by zaza88i** [overextended/ox_inventory#1838]
SCRIPT ERROR: @ox_inventory/modules/bridge/esx/server.lua:22: ox_inventory requires a ESX Legacy v1.6.0 or above, refer to the documentation.

**Comment by Sugarhillny** [comment]
que esta pasando con el ox_inventory para esx

**Comment by Sugarhillny** [comment]
esta dando un grave error nose por que

**Comment by Sugarhillny** [comment]
SCRIPT ERROR: @ox_inventory/modules/bridge/esx/server.lua:20: ox_inventory requires Config.CustomInventory to be set to "ox" in the ESX Config este error

**Comment by Sugarhillny** [comment]
![Sin título](https://github.com/user-attachments/assets/ef188d8d-c35e-4ca8-87d4-13d6bc32f8a1)

**Comment by zaza88i** [comment]
im getting same issue

**Comment by Sugarhillny** [comment]
ok

**Comment by Akku67** [comment]
[ script:ox_inventory] SCRIPT ERROR: @ox_inventory/modules/bridge/esx/server.lua:20: ox_inventory requires Config.CustomInventory to be set to "ox" in the ESX Config. 



help

**Comment by thelindat** [comment]
Read it. It's in plain English and telling you what to do. This is not an issue with ox_inventory, nor does anything need to be changed in ox_inventory. Once again - read what that message is telling you to do.

**Comment by Akku67** [comment]
if customInventory ~= nil and customInventory ~= "ox" then
        error('ox_inventory requires Config.CustomInventory to be set to "ox" in the ESX Config.')
    elseif ESX.CreatePickup then
		error('ox_inventory requires a ESX Legacy v1.6.0 or above, refer to the documentation.')
	end

**Comment by thelindat** [comment]
I am _pleading_ for you to _read_ and follow the instructions provided to you.

**Comment by Akku67** [comment]
please help

**Comment by thelindat** [comment]
_Why_ are you postinng here **still**? I _cannot_ do anything for you. There is _no problem in this script. There is no problem in any script_.

The _only problem_ is in your head.

**Comment by Akku67** [comment]
error('ox_inventory requires a ESX Legacy v1.6.0 or above, refer to the documentation.')

**Comment by MrBusinessGRx** [comment]
> _Why_ are you posting here **still**? I _cannot_ do anything for you. There is _no problem in this script. There is no problem in any script_.
> 
> The _only problem_ is in your head.

I'd appreciate if you could help us with that error on what to do to fix it instead of saying what the error says! We know what the error says, how do we update to fix it? i literally installed https://runtime.fivem.net/artifacts/fivem/build_server_windows/master/10188-607fd2ca5265f18c16fbbe0da3682820cf21eeb6/server.7z which should be up to date and still doesnt work. UNLESS, there's another way to fix it and im doing it wrong. Thanks

**Comment by MrBusinessGRx** [comment]
Hey, deleted the previous message as it wasnt pretty clear.
Could you please tell me how do I update esx legacy? Installed build 10188 still same error.
![image](https://github.com/user-attachments/assets/dd084165-e5ab-465c-b114-42f4f402a464)

**Comment by zaza88i** [comment]
what these ppl don't get is we not slow u can change all u want it still shows up. bro its the script just ditch it. just get another something wrong with the code

**Comment by thelindat** [comment]
> what these ppl don't get is we not slow
> something wrong with the code. Not ox but script u using

[This guy](https://github.com/overextended/ox_inventory/issues/1846#issuecomment-2518434396) responded to one of the half a dozen duplicate issues about this topic, with exact instructions and screenshots to correctly configure ESX 1.11+ to support ox_inventory. The next user still failed to modify it.

![392549772-0f814a35-8768-49bc-847d-bc5c7d6822bb](https://github.com/user-attachments/assets/fbd8f8ff-ef6a-4592-8d30-a71df1270bc0)
